src/main/resources/static/css/module.css

```diff
@@ -0,0 +1,190 @@
+/* Home Page Modules */
+
+.entryDate{
+    padding-top: 10px;
+    padding-left: 20px;
+    font-weight: bold;
+    text-decoration: underline;
+}
+
+#entry-line{
+    outline: solid;
+    outline-color: #19535F;
+    background-color: floralwhite;
+    margin: 1%;
+    padding: 1%;
+    margin-top: 3%;
+}
+
+#newEntry{
+    margin-top: 20px;
+    text-align: center;}
+
+#journal-svg{
+    display: inline-block;
+    width: 2%;
+}
+
+#journal-entry{
```

Comment:
Recommendation:  Try to limit how often you use id's in your css.  Generally they're only used for top level layout elements, or specialty pieces ("this one off thing needs an id because of x").